examples/array_of_notes/sketch.js

```diff
@@ -0,0 +1,45 @@
+var synth;
+var songStarted = false;
+var song = [ 
+  // Note pitch, velocity (between 0-1), start time (s), note duration (s)
+  {pitch:'E4', velocity:1, time:0, duration:1},
+  {pitch:'D4', velocity:1, time:1, duration:1},
+  {pitch:'C4', velocity:1, time:2, duration:1},
+  {pitch:'D4', velocity:1, time:3, duration:1},
+  {pitch:'E4', velocity:1, time:4, duration:1},
+  {pitch:'E4', velocity:1, time:5, duration:1},
+  {pitch:'E4', velocity:1, time:6, duration:1},
+  // Rest indicated by offset in start time
+  {pitch:'D4', velocity:1, time:8, duration:1},
+  {pitch:'D4', velocity:1, time:9, duration:1},
+  {pitch:'E4', velocity:1, time:10, duration:1},
+  {pitch:'D4', velocity:1, time:11, duration:1},
+  // Chord indicated by simultaneous note start times
+  {pitch:'C4', velocity:1, time:12, duration:2},
+  {pitch:'E4', velocity:1, time:12, duration:2},
+  {pitch:'G4', velocity:1, time:12, duration:2},
+];
+
+function setup() {
+  textAlign(CENTER, CENTER);
+  synth = new p5.PolySynth();
+}
+
+function draw() {
+  background(255);
+  if (songStarted) {
+    text('song started', width/2, height/2);
+  } else {
+	  text('click to play song', width/2, height/2);
+  }
+}
+
+function touchStarted() {
+  if (!songStarted) { // Only play once
+    for (var i=0; i<song.length; i++) {
```

Comment:
I think we could introduce the SoundLoop here. That way the song could be re-started by resetting the index, or paused / stopped.

Reply:
I decided to introduce the SoundLoop in a separate example, because it makes playback somewhat more complicated than necessary depending on what the user needs. So there are now two examples, `array_of_notes` and `array_of_notes_soundloop` which illustrate two possible ways to play music from an array.